### DISCUSSION TITLE
[1710100] 黄老饼梦游惊奇 [本体]

### DISCUSSION THREAD

**Discussion by ZapRill** [pjy612/SteamManifestCache#240] [Share]
`appid:1710100`
keys:

```
1710101;696a45c513e6fe62f82449905bfb2b7684e227eec359f11993833a62390b0a4e
```

CODE:CF0CD396F53F7629
email:uzs21261904200403@gmail.com
[清单.zip](https://github.com/pjy612/SteamManifestCache/files/13817199/default.zip)

**Comment by pjy612** [comment]
未收录 麻烦附带清单

**Reply by ZapRill** [reply]
已附清单

**Reply by pjy612** [reply]
查收
